sass/themes/donate/sr18/_donate-sr18-styles.scss

```diff
@@ -5,3 +5,48 @@ body.payin {
 .header {
   display: none;
 }
+
+/* Thank You page */
+#promo-header--success {
+  br {
+    display: inline;
+  }
+
+  h1, h2, h3 {
+    text-transform: none;
+  }
```

Comment:
should we override the letter-spacing on h2?

Reply:
As discussed, we'll be adhering to PL styles rather than the design-made tweaks